server/mdm/android/tests/http.go

```diff
@@ -0,0 +1,49 @@
+package tests
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/test/httptest"
+	"github.com/go-json-experiment/json"
+	"github.com/stretchr/testify/require"
+)
+
+func (ts *WithServer) DoJSON(verb, path string, params interface{}, expectedStatusCode int, v interface{}, queryParams ...string) {
```

Comment:
That's fine for now, but if we reorganize our `service` integration tests into sub-packages, we should have a common tester helper package for this kind of methods.